Features/CaseSearch/test_pages/casesearch_page.py

```diff
@@ -163,10 +163,11 @@ def check_value_on_case_detail(self, search_property, expected_value):
         assert self.is_visible_and_displayed(value)
 
     def check_case_claim_case_type(self, claimed_case_name, claimed_user):
-        self.select_by_text(self.combox_select_element, "commcare-case-claim")
+        self.select_by_text(self.case_type_select, "commcare-case-claim")
         self.wait_to_clear_and_send_keys(self.report_search, claimed_case_name)
         self.wait_to_click(self.report_apply_filters)
         claim_case_type = (By.XPATH, self.commcare_case_claim_case.format(claimed_case_name, claimed_user))
+        print(claim_case_type)
```

Comment:
Is this required?

Reply:
Not required. Removed all unwanted print statements in https://github.com/dimagi/dimagi-qa/pull/156/commits/f71834514827878af718a1684b9d25f24c49da76